coins.json

```
@@ -1543,5 +1543,29 @@
       "clientPublic": "https://bsc-dataseed1.binance.org",
       "clientDocs": "https://eth.wiki/json-rpc/API"
     }
+  },
+  {
+    "id": "matic",
+    "name": "Matic",
```

From https://coinmarketcap.com/currencies/polygon/
```suggestion
    "name": "Polygon",
```